pkg/output/output.go

```diff
@@ -0,0 +1,67 @@
+//
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package output
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	Format  string
+	Formats = map[string]struct{}{
+		"yaml": {},
+	}
+)
+
+type YamlCaseResult struct {
+	Passed  []string
+	Failed  []string
+	Skipped []string
+}
+
+func HasFormat() bool {
+	_, ok := Formats[Format]
+	return ok
+}
+
+func PrintResult(caseRes []*CaseResult) {
+	if Format == "yaml" {
+		PrintResultInYAML(caseRes)
+	}
```

Comment:
use `switch` statement?

Reply:
I think 'switch' is better, but if I use 'switch', the go-lint will go wrong. The go-lint suggests me to use 'if' statement.